.github/workflows/build-latest.yaml

```diff
@@ -13,17 +13,35 @@ jobs:
       - name: Tests
         run: make check
   build:
+    permissions:
+      packages: write
+      contents: read
     name: Docker build
     runs-on: ubuntu-latest
     needs:
       - tests
     steps:
+      - name: Set up Docker Buildx with QEMU
+        uses: docker/setup-buildx-action@v3
+      - uses: docker/setup-qemu-action@v3
       - uses: actions/checkout@v4
+      - name: Convert relevant environment variables
+        id: convert
+        run: |
+          echo "REPO=$(echo '${{ github.repository }}' | tr '[:upper:]' '[:lower:]')" >> $GITHUB_ENV
+          echo "TAG_VERSION=$(echo '${{ github.ref_name }}' | sed 's/^v//')" >> $GITHUB_ENV
       - name: Login to GitHub Container Registry
         uses: docker/login-action@v3
         with:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
-      - run: docker build . -t `echo ghcr.io/${{ github.repository }}:latest | tr '[:upper:]' '[:lower:]'`
-      - run: docker push `echo ghcr.io/${{ github.repository }}:latest | tr '[:upper:]' '[:lower:]'`
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v6
+        with:
+          file: ./Dockerfile
+          platforms: linux/amd64,linux/arm64
```

Comment:
This builds images for two architectures, which is great!
Does it mean that it performs testing under two archs as well?

Reply:
> Does it mean that it performs testing under two archs as well?

Not at the moment. I was unsure if the test mentioned in the `tests` job is sufficient for this specific use-case. 

I can add both architectures for the `tests` job, if `make check` is sufficient testing for the Docker images?